libs/community/langchain_community/vectorstores/vectara.py

```diff
@@ -66,21 +73,47 @@ class VectaraQueryConfig:
     score_threshold: minimal score threshold for the result.
         If defined, results with score less than this value will be
         filtered out.
-    n_sentence_context: number of sentences before/after the matching segment
+    n_sentence_before: number of sentences before the matching segment
```

Comment:
some of these attribute changes are breaking [removals + changed defaults]. Is there a way to do this in a nonbreaking way?

Reply:
Great point and thanks for pointing that out. 
I made some changes to make everything backwards compatible. In the process also added test_cases to also test it works in the new form and in the old form, and also updated default values to ensure nothing is a breaking change.